mathics/builtin/structure.py

```diff
@@ -1276,5 +1281,10 @@ class ByteCount(Builtin):
 
     def apply(self, expression, evaluation):
         'ByteCount[expression_]'
-
-        return Integer(count_bytes(expression))
+        if bytecount_support == False:
```

Comment:
I think it's just PyPy. Jython too [supports](https://mail.python.org/pipermail/jython-checkins/2016-January/001669.html) sys.getsizeof().